pkg/controller/controller.go

```diff
@@ -274,6 +275,19 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 	var err error
 	var privKey []byte
 	var apicCert []byte
+
```

Comment:
Please move this to cmd/controller/main.go. If you add --version as a cli flag there, you can also eliminate the shell script. Instead, you could just invoke controller --version which would just print the version and exit.

Reply:
Thanks for the suggestion, made life easier. Removed the script and added the --version cli flag to aci-containers-controller